File: src/android/framework/PathSegment.java
//-------------------------------------------------------------------------------------------------------
// Copyright (C) Microsoft. All rights reserved.
// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
//-------------------------------------------------------------------------------------------------------
package Windows.UI.Xaml.Controls;

public class PathSegment {
    boolean _isSmoothJoin;
    boolean _isStroked;

    boolean getIsSmoothJoin() {
        return _isSmoothJoin;
    }
    public void setIsSmoothJoin(boolean isSmoothJoin) {
        _isSmoothJoin = isSmoothJoin;
    }

    boolean getIsStroked() {
        return _isStroked;
    }
    public void setIsStroked(boolean isStroked) {
        _isStroked = isStroked;
    }
}

File: www/framework/PathSegment.js
//-------------------------------------------------------------------------------------------------------
// Copyright (C) Microsoft. All rights reserved.
// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
//-------------------------------------------------------------------------------------------------------

//
// Base class for a segment of a PathFigure.
//
function PathSegment() {
    ace.UIElement.call(this, "Windows.UI.Xaml.Media.PathSegment");
};

// Inheritance
PathSegment.prototype = Object.create(ace.NativeObject.prototype);

module.exports = PathSegment;
